modules/jaeger-config-sampling.adoc

```diff
@@ -27,25 +27,30 @@ Jaeger libraries support the following samplers:
 
 * *Remote* - The sampler consults the Jaeger agent for the appropriate sampling strategy to use in the current service. This allows controlling the sampling strategies in the services from a central configuration in the Jaeger backend.
 
-.Jaeger sampling parameters
+.Parameters used by the Operator to define Jaeger sampling
 [options="header"]
 [cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
+|sampling:
+  type:
+|Sampling strategy to use. (See descriptions above.)
+|Valid values are `const`, `probabilistic`, `ratelimiting`, and `remote`.
+|`remote`
+|===
```

Comment:
In sampling there is no parameter used by the operator, all the parameters should come under `options`
and supported `type`: `probabilistic` and `ratelimiting`
Input format should be as follows

@jpkrohling Could you please confirm?

Source: https://www.jaegertracing.io/docs/1.22/sampling/#collector-sampling-configuration
```yaml
spec:
  sampling:
    options:
      default_strategy:
        type: probabilistic
        param: 0.5
      service_strategies:
        - service: foo
          type: probabilistic
          param: 0.8
          operation_strategies:
            - operation: op1
              type: probabilistic
              param: 0.2
            - operation: op2
              type: probabilistic
              param: 0.4
        - service: bar
          type: ratelimiting
          param: 5
```

if non supplied by the user the default values:
```yaml
spec:
  sampling:
    options:
      default_strategy:
        type: probabilistic
        param: 1
```

Reply:
I think you're right here: https://github.com/jaegertracing/jaeger-operator/blob/028965e53962950c17e93f60d582613dd9df7b12/pkg/apis/jaegertracing/v1/jaeger_types.go#L274

All those attributes should be under options.